src/v2/Apps/Auctions/Components/StandoutLots/StandoutLots.tsx

```diff
@@ -0,0 +1,93 @@
+import { AuthContextModule, clickedArtworkGroup } from "@artsy/cohesion"
+import { Shelf, Spacer, Text } from "@artsy/palette"
+import { graphql } from "lib/graphql"
+import React from "react"
+import { createFragmentContainer } from "react-relay"
+import { useTracking } from "react-tracking"
+import { ShelfArtworkFragmentContainer } from "v2/Components/Artwork/ShelfArtwork"
+import { useAnalyticsContext } from "v2/System"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { StandoutLots_viewer } from "v2/__generated__/StandoutLots_viewer.graphql"
+import { tabTypeToContextModuleMap } from "../../Utils/tabTypeToContextModuleMap"
+
+export interface StandoutLotsProps {
+  viewer: StandoutLots_viewer
+}
+
+const StandoutLots: React.FC<StandoutLotsProps> = ({ viewer }) => {
+  const { trackEvent } = useTracking()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const contextModule = tabTypeToContextModuleMap.standoutLots as AuthContextModule
+
+  const nodes = extractNodes(viewer.standoutLotsConnection)
+
+  if (nodes.length === 0) {
+    return null
+  }
+
+  return (
+    <>
+      <Text as="h3" variant="lg" color="black100">
+        Standout Lots{" "}
+        <sup>
+          <Text as="span" variant="xs" color="brand">
```

Comment:
So we've got a component for this that handles font-sizes and line heights correctly — import and use `Sup` from palette

Reply:
(And you'd use it in place of both `sup` and `Text` (— you can pass it color="brand")

Reply:
Does `Sup` handle the variant and span portions of it?

Reply:
Yeah, it adjusts based on the parent variant. You wouldn't need to use `as` either, in this case.

Reply:
Aight, cool. Updated.